mozilla_django_oidc/auth.py

```diff
@@ -279,6 +279,12 @@ def get_userinfo(self, access_token, id_token, payload):
             proxies=self.get_settings("OIDC_PROXY", None),
         )
         user_response.raise_for_status()
+
+        if user_response.headers.get("content-type", "").startswith("application/jwt"):
```

Comment:
MIME types are case-insensitive you may want to be also?

> MIME types are case-insensitive but are traditionally written in lowercase. The parameter values can be case-sensitive.
> https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/MIME_types

```suggestion
        if user_response.headers.get("content-type", "").lower().startswith("application/jwt"):
```